dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/newTimer/TestEjbTimerServlet.java

```diff
@@ -0,0 +1,48 @@
+ /*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.ejb.apps.newtimer;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import javax.ejb.EJB;
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import componenttest.app.FATServlet;
+import org.junit.Test;
+
+@WebServlet("/Timer")
+public class TestEjbTimerServlet extends FATServlet {
+    private static final long serialVersionUID = 1L;
+    @EJB
+    EjbSessionBeanLocal bean;
+
+    @Test
+    public void testEjbTimers() throws ServletException, IOException {
+        bean.initTimer();
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            assertTrue("The timer exploded", false);
```

Comment:
1. There's a `fail` method, which is preferable to using `assertTrue(false)`.
2. It's usually makes it easier to debug the problem if you allow the original exception to be thrown, rather than catching it and failing the test.
   a. If you do want to add a message to say what happened, you can do `throw new AssertionError("my message", e);` to add your message while preserving the original exception.

Reply:
This still needs fixed. This should do it:
```
throw new AssertionError("The timer exploded", e);
```